EnglishFilter/sections/foreign.txt

```diff
@@ -4156,6 +4156,8 @@ naekranie.pl#%#AG_abortInlineScript(/loadData/, '$');
 xboxforum.pl##.adsbygoogle
 ! https://github.com/AdguardTeam/AdguardFilters/issues/36012
 ddwloclawek.pl,ddtorun.pl##.topLayer
+! https://github.com/AdguardTeam/AdguardFilters/issues/37065
+wylecz.to#?#.row > div > div > div > div:contains("REKLAMA")
```

Comment:
It seems that some ads are not blocked by this rule - https://wylecz.to/badania-laboratoryjne/lipidogram/
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/61456251-df9e5f00-a965-11e9-9b32-ad25f887a013.png)

</details>

This rule should be okay:
`wylecz.to#?#.row div[class][style*="border-bottom:"][style*="max-height:"]:has(> div[style]:contains(REKLAMA))`